scripts/util/keystore-cert-import

```diff
@@ -38,9 +38,6 @@ if [ -z "${KS_PASS// }" ]; then
     exit 1
 fi
 
-# Use a new keystore file
-NEW_KS_FILE="$KS_FILE.new"
```

Comment:
The script was create to be idempotent for jks creation and cert import. I'll have to test concurrent cases and see if this script would pass/fail. I'll test and review today and keep you posted @nvazquez thanks.

Reply:
I'll also need to test for certificate renewal.